Exercise-HTML-CSS-1/js/script.js

```diff
@@ -0,0 +1,44 @@
+var express = require("express");
```

Comment:
I suggest you to remove the content of this file. We'll eventually move to Node and Mongo, but not yet!